feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog-event.listener.ts

```diff
@@ -61,6 +82,45 @@ export class AddedToCartDialogEventListener implements OnDestroy {
     }
   }
 
+  /**
+   * Opens modal after addToCart has been successfully performed.
+   * @param successEvent Signals that the backend call succeeded.
+   */
+  protected openModalAfterSuccess(
+    successEvent: CartAddEntrySuccessEvent
+  ): void {
+    const addToCartData: AddedToCartDialogComponentData = {
+      productCode: successEvent.productCode,
+      quantity: successEvent.quantity,
+      pickupStoreName: successEvent.pickupStore,
+      addedEntryWasMerged: this.calculateEntryWasMerged(successEvent),
+    };
+
+    const dialog = this.launchDialogService.openDialog(
+      LAUNCH_CALLER.ADDED_TO_CART,
+      undefined,
+      undefined,
+      addToCartData
+    );
+
+    if (dialog) {
+      dialog.pipe(take(1)).subscribe();
+    }
+  }
+
+  protected calculateEntryWasMerged(
+    successEvent: CartAddEntrySuccessEvent
+  ): boolean {
+    // In case quantityAdded zero, the system could have run into stock issues.
```

Comment:
From unit tests and implementation I understood that in case of stock issues the property `quantityAdded` is `undefined` (not `0`).

So what about updating the comment:
```suggestion
    // In case `quantityAdded` is `undefined`, the system could have run into stock issues.
```

Btw. IMHO we could turn this inline comment in to a fully-fledged JSDoc of this method.